python-project-template/.github/workflows/{% if preferred_linter != 'none' %}pre-commit-ci.yml{% endif %}.jinja

```diff
@@ -0,0 +1,34 @@
+# This workflow runs pre-commit hooks on pull requests to enforce coding style.
+# To ensure correct configuration, please refer to:
+#   https://lincc-ppt.readthedocs.io/en/latest/practices/ci_precommit.html
+
+name: Run pre-commit hooks
+
+on:
+  pull_request:
+
+jobs:
+  pre-commit-ci:
+    runs-on: ubuntu-latest
+    env: 
+      SKIP: "check-lincc-frameworks-template-version,pytest-check,no-commit-to-branch,validate-pyproject,check-added-large-files,sphinx-build"
```

Comment:
I don't like having this huge skip environment variable here. Any ideas on how I could improve this?

Reply:
Is the concern just that it's concatenating all the names together, and looks gross?

I agree that it looks gross, but it's probably ok. I'm not sure if the behavior of `env:` is to set or append. If append, you could split it up like (including the commas at the end of the lines):

```
env:
  SKIP: "check-lincc-frameworks-template-version,"
  SKIP: "pytest-check,"
  SKIP: "no-commit-to-branch,"
  SKIP: "validate-pyproject,"
  ...
```

Or you can try some github action language operators like:

```
env:
  SKIP: join(["check-lincc-frameworks-template-version",
                  "pytest-check",
                  "no-commit-to-branch",
                  "validate-pyproject",
                  "check-added-large-files",
                  "sphinx-build"], ',')
```

Reply:
Either of @delucchi-cmu suggestions look good to me, but I would be ok with just leaving it as is. It's not so long that a user couldn't read through it and pretty much understand what's going on.

Reply:
Alright, if it's no hassle I left as is!